core/src/main/scala/chisel3/probe/package.scala

```diff
@@ -68,10 +68,21 @@ package object probe extends SourceInfoDoc {
     }
   }
 
+  /** Pad Data if it supports padding */
+  private def padData[T <: Data](data: T, width: Int)(implicit sourceInfo: SourceInfo): T = {
+    if (data.width.get > width) {
+      Builder.error(s"Data width ${data.width.get} is larger than $width.")
+    }
```

Comment:
What if the width of the data is unknown? Does firtool accept forcing from a Wire that may have inferred width?

If that's illegal, then there needs to be a specific check and error rather than just a `.get`.

Reply:
i added in another check for a known width before this check, does that suffice?